src/TableView.elm

```diff
@@ -115,17 +120,19 @@ viewDate model numColumns dateWithSessions =
             dateWithSessions.sessions
                 |> List.concatMap
                     (\s ->
-                        [ DateUtils.timeOfDayToTime dateWithSessions.date s.startTime
-                        , DateUtils.timeOfDayToTime dateWithSessions.date s.endTime
+                        [ { delimiter = DateUtils.timeOfDayToTime dateWithSessions.date s.startTime
+                          , displayTime = s.startTime
+                          }
+                        , { delimiter = DateUtils.timeOfDayToTime dateWithSessions.date s.endTime, displayTime = s.endTime }
                         ]
                     )
-                |> Utils.dropDuplicates
-                |> List.sort
+                |> List.Extra.uniqueBy (\t -> t.delimiter)
+                |> List.sortBy .delimiter
 
         firstTime =
             timeDelimiters
                 |> List.head
-                |> Maybe.withDefault 8
+                |> Maybe.withDefault { delimiter = 8, displayTime = { hour = 0, minute = 0 } }
```

Comment:
Does this still display 8am as a default timetable start time when there are no sessions?

Reply:
Ah is that the expected behaviour? I was a bit confused by the 8. Will change

Reply:
![screen shot 2018-01-11 at 15 52 19](https://user-images.githubusercontent.com/11833296/34846832-701e78f4-f6e7-11e7-9530-9013d80deb61.png)

I don't think this is the current behaviour? Or have I misunderstood? Please see above

Reply:
Have changed the hour to 8 though anyway